### ISSUE TITLE
cannot find naddr event

### ISSUE THREAD

**Issue by melvincarvalho** [DanConwayDev/gitworkshop#5]
I'm trying to display the following naddr

naddr1qqr8xet5dp6k2qgswaehxw309ah8qatz9e5kuen09upzpph6xk3aymeupueju05pyssq2l8sum8ejlzmu8z5spn2p8rrfkh7qvzqqqrhny3mk22n

https://gitworkshop.dev/r/naddr1qqr8xet5dp6k2qgswaehxw309ah8qatz9e5kuen09upzpph6xk3aymeupueju05pyssq2l8sum8ejlzmu8z5spn2p8rrfkh7qvzqqqrhny3mk22n/proposals

it decodes properly

$ nak decode naddr1qqr8xet5dp6k2qgswaehxw309ah8qatz9e5kuen09upzpph6xk3aymeupueju05pyssq2l8sum8ejlzmu8z5spn2p8rrfkh7qvzqqqrhny3mk22n

```json
{
  "pubkey": "86fa35a3d26f3c0f332e3e812420057cf0e6cf997c5be1c548066a09c634dafe",
  "kind": 30617,
  "identifier": "sethue",
  "relays": [
    "wss://npub.info/"
  ]
}
```

and festches an event

$ nak fetch naddr1qqr8xet5dp6k2qgswaehxw309ah8qatz9e5kuen09upzpph6xk3aymeupueju05pyssq2l8sum8ejlzmu8z5spn2p8rrfkh7qvzqqqrhny3mk22n
```json
{
  "id": "02bff54be7dbf3da1ea2e3da91cac37ca6acc2297f40eddc499a21dfda4df01b",
  "pubkey": "86fa35a3d26f3c0f332e3e812420057cf0e6cf997c5be1c548066a09c634dafe",
  "created_at": 1720629803,
  "kind": 30617,
  "tags": [
    [
      "d",
      "sethue"
    ],
    [
      "relays",
      "wss://npub.info/"
    ],
    [
      "c",
      "txo:tbtc4:e0a84ec838712ddec78c57a7dc1ce65b777cbf58bfa861f289e3533a60399493:0"
    ],
    [
      "clone",
      "http://git.melvincarvalho.com/smart/.git"
    ]
  ],
  "content": "genesis: smart 32",
  "sig": "2eb8782cf85b711656af367b807b00a26dbb522a68add2a4ff3db7030e7b6f044d3cdf35866b3a2cc3f012ff409e93f69dc15fb750fde35cfd11c6715581a388"
}
```

but front end cant display:

![image](https://github.com/DanConwayDev/gitworkshop/assets/65864/9902f40e-92b7-401c-9020-de7b66e86303)

Any ideas?

**Comment by DanConwayDev** [comment]
Hello Melvin,

We are not using github to track issues. Please can you create an issue via nostr? https://gitworkshop.dev/r/naddr1qq9kw6t5wahhy6mndphhqqgdwaehxw309ahx7uewd3hkcq3q5qydau2hjma6ngxkl2cyar74wzyjshvl65za5k5rl69264ar2exsxpqqqpmejfryfcu/issues

**Comment by DanConwayDev** [comment]
Although I have just fixed it in commit 46d94eaac743f1137521c19762f372a66d72228e.
It wasn't using the relays embedded in the naddr
